src/ngame/client/client.cljs

```diff
@@ -79,17 +80,26 @@
     (set! (.-y game-object) y)
     (add-player-to-map! player-id player-pos game-object)))
 
+(defn clean-up-player!
+  [player-id]
+  (let [player-to-clean-up (get player-map player-id)
+        game-object (get player-to-clean-up :game-object)]
+    (if (not (nil? game-object))
```

Comment:
super-nit: Did you see `if-let`? Might save a few keystrokes.

Reply:
All about savings those few keystrokes! 🙏